### DISCUSSION TITLE
github Timeout Errors

### DISCUSSION THREAD

**Discussion by atvcaptain** [community/community#75000] [Repositories]
### Select Topic Area

General

### Body

Access from Germany RZ Hetzner
timeout errros

git -c gc.autoDetach=false -c core.pager=cat ls-remote https://github.com/DeadEyE-OpenATV/BlackSpirit.HD.git
fatal: unable to access 'https://github.com/DeadEyE-OpenATV/BlackSpirit.HD.git/': Failed to connect to github.com port 443 after 130207 ms: Connection timed out

**Comment by atvcaptain** [comment]
there are always other github gits affected where there are timeout erros, it has been like this for 2-3 hours

**Comment by jbleyel** [comment]
It seams that this problem occurs every Friday and probably only in Europe.
This is not acceptable!!

**Comment by jbleyel** [comment]
Similar issue on Friday -> 
https://github.com/orgs/community/discussions/70832
